### PR TITLE
Add fixture data for mongodb and use standart mongo image

### PR DESCRIPTION
**Pull request by cihangir**

### CODE REVIEW

**Review by szkl — APPROVED** [review]

File: workers/social/lib/social/models/log/index.coffee [hunk 1]
```diff
@@ -1,8 +1,10 @@
 { Module } = require 'jraphical'
+KONFIG     = require 'koding-config-manager'
 
 module.exports = class JLog extends Module
 
-  TRY_LIMIT_FOR_BLOCKING  = 5
+  # do not make QA's waiting for 5 mins. We test this feature only test on sandbox
+  TRY_LIMIT_FOR_BLOCKING  = if KONFIG.environment is 'sandbox' then 1 else 5
```
Comment (cihangir) [inline workers/social/lib/social/models/log/index.coffee]: cc/ @diclee

File: config/generateRunFile.coffee [hunk 2]
```diff
@@ -190,18 +176,13 @@ generateDev = (KONFIG, options) ->
       echo "  run logs                  : to see all workers logs"
       echo "  run log [worker]          : to see of specified worker logs only"
       echo "  run buildservices         : to initialize and start services"
-      echo "  run resetdb               : to reset databases"
```
Comment (gokmen) [inline config/generateRunFile.coffee]: can we keep this? it's very handy.

File: scripts/bootstrap-container [hunk 3]
```diff
@@ -16,9 +16,8 @@ function build() {
 
 function run_backend() {
   go/build.sh
-  ./run exec scripts/check-service-connectivity.sh || exit 255
-  ./run migrate up
-  nginx -c $(pwd)/nginx.conf
```
Comment (szkl) [inline scripts/bootstrap-container]: How does nginx is started?

Comment (cihangir) [inline scripts/bootstrap-container]: https://github.com/koding/koding/pull/9291/files#diff-3cf08f99982d75678171b65886c3a2b2R36

Comment (szkl) [inline scripts/bootstrap-container]: buildservices is not used for containers.

Comment (cihangir) [inline scripts/bootstrap-container]: good catch, added back

File: package.json [hunk 4]
```diff
@@ -71,7 +71,7 @@
     "mime": "1.2.x",
     "minimist": "0.0.8",
     "mkdirp": "0.3.2",
-    "mongodb": "1.4.14",
+    "mongodb": "1.4.18",
```
Comment (cihangir) [inline package.json]: i am feeling adventurous 😎

Comment (szkl) [inline package.json]: Good luck. Let me know if you get any failures in unit tests.

**Comment by cihangir** [comment]
🏁

**Pull request merged**